src/main/java/lcduke/Duke.java

```diff
@@ -0,0 +1,132 @@
+package lcduke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.text.ParseException;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/** Ths creates a Duke object.
+ */
+
+public class Duke extends Application {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+
+    public Duke(){
+        this("/Users/liuchao/duke/src/main/");
+    }
+
+    /** This is the constructor to create the Deadline Object.
+     *
+     * @param filePath File path of user's hard disk.
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load(), storage.getStorageNo());
+        } catch (FileNotFoundException | ParseException e) {
+            ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+    public void run(){
+        String userInput = ui.input();
+        Parser newInput = new Parser(userInput);
+
+        while(!userInput.equals("bye")){
+            if(newInput.getIsProblem()) {
```

Comment:
Using a switch case would make your code neater and more readable. It'll also make your logic looks more organized